Src/BootCamp.Chapter/Person.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Person
+    {
+        private readonly string _name;
+        private readonly decimal[] _balances;
+
+        public Person(string name, decimal[] balances)
+        {
+            _name = name;
+            _balances = balances;
+        }
+
+        public string GetName()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetName' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)